Restore.cmd

```diff
@@ -7,6 +7,7 @@ set NuGetAdditionalCommandLineArgs=-verbosity quiet -configfile "%RoslynRoot%nug
 :ParseArguments
 if /I "%1" == "/?" goto :Usage
 if /I "%1" == "/clean" set RestoreClean=true&&shift&& goto :ParseArguments
+if /I "%1" == "/fast" set RestoreFast=true&&shift&& goto :ParseArguments
```

Comment:
It's for iterating locally a bit faster.  I'm having to restore a lot during this change and full restore was taking up too much time.  Needed to trim it down. 

I realize it's unsafe and shouldn't be done in a real build environment.  For local iteration though I'm willing to absorb some risk.